### PR TITLE
Merge develop into main

### LINKED ISSUE #37: Remove profile limit
Remove the limit to the number of profiles that can be added to the application. If the profile limit is maxed out to 256 profiles, please include an error message if a user exceeds that limit.

### PR DESCRIPTION
**Pull request by DSMatusevich**
**Description**:
Version 0.6.0 release

Closes #37 #128 #173 #174 #180

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by qnswirlds — COMMENTED** [review]
Please resolve conflicts first.

**Comment by DSMatusevich** [comment]
> Please resolve conflicts first.

Done. Please check again

File: tools-ui/src/main/java/com/hedera/hashgraph/client/ui/utilities/TimeFieldSet.java [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+	TextField hours;
+	TextField minutes;
+	TextField seconds;
+	TextField nanos;
+
+	HBox timeZoneBox;
+	TimeZone timeZone;
+
+	Label utcLabel;
+	Label errorLabel;
```
Comment (qnswirlds) [inline tools-ui/src/main/java/com/hedera/hashgraph/client/ui/utilities/TimeFieldSet.java:52]: I probably did not review the PR for this change. But these fields should come with `private` right?

Comment (DSMatusevich) [reply tools-ui/src/main/java/com/hedera/hashgraph/client/ui/utilities/TimeFieldSet.java:52]: Gosh, yes! I don't know how that escaped me. I will make the change now. Great catch, thanks

**Pull request closed without merging**